ci.nix

```diff
@@ -43,7 +43,9 @@
     # We need to use the actual nixpkgs version we're working with here, since the values
     # of 'lib.systems.examples' are not understood between all versions
     let lib = nixpkgs.lib;
-    in lib.optionalAttrs (system == "x86_64-linux" && (nixpkgsName == "R2009" || (compiler-nix-name != "ghc8101" && compiler-nix-name != "ghc8102" && compiler-nix-name != "ghc8102-experimental"))) {
+    in lib.optionalAttrs (nixpkgsName == "R2003" && (compiler-nix-name == "ghc865" || compiler-nix-name == "ghc884")) {
+    inherit (lib.systems.examples) ghcjs;
+  } // lib.optionalAttrs (system == "x86_64-linux" && (nixpkgsName == "R2009" || (compiler-nix-name != "ghc8101" && compiler-nix-name != "ghc8102" && compiler-nix-name != "ghc8102-experimental"))) {
```

Comment:
Don't we have some `elem` function in nixpkgs? this chain of `&&` + `!=` seems rather hard to read.